rita_client/src/dashboard/interfaces.rs

```diff
@@ -156,204 +156,238 @@ pub fn ethernet2mode(ifname: &str, setting_name: &str) -> Result<InterfaceMode,
     })
 }
 
-fn set_interface_mode(iface_name: &str, mode: InterfaceMode) -> Result<(), RitaClientError> {
+fn set_interface_mode(
```

Comment:
If it's practical it would be better to have a new function multiset_interfaces() or somthing like that that contains the new validation logic running against the arrays of ports to toggle and then keep set_interface_mode and set_ethernet_mode as functions that operate on a single interface at a time.

This makes the multiple port toggling easier to read and reason about in my opinion.

Reply:
fixed